apps/backend/scripts/fetch-term-data.ts

```diff
@@ -9,8 +9,11 @@ import { dirname, join } from 'node:path';
 import { fileURLToPath } from 'node:url';
 
 import { CalendarTerm } from '@packages/antalmanac-types';
+import { backendEnvSchema } from 'src/env';
 
-const apiKey = process.env.ANTEATER_API_KEY;
+import 'dotenv/config';
+
+const env = backendEnvSchema.parse(process.env);
```

Comment:
```suggestion
const env = aapiEnvSchema.parse(process.env);
```

This is the only one that you need, and it should allow the workflow to run properly